python/hail/context.py

```diff
@@ -465,6 +471,74 @@ def import_vcf(self, path, force=False, force_bgz=False, header_file=None, npart
         return VariantDataset(self, jvds)
 
     @handle_py4j
+    def import_vcf_generic(self, path, force=False, force_bgz=False, header_file=None, npartitions=None,
```

Comment:
I read `import_generic_vcf` as importing some thing called a Generic VCF.

I don't have a strong feeling/intuition, I'd go with whatever our users would like the most.